pawn/Elements/AdministratorCommands.pwn

```diff
@@ -972,6 +972,11 @@ lvp_show(playerId, params[]) {
         showInfo = true;
         format(g_message, sizeof(g_message), "Save money and stats! Register your nickname at www.sa-mp.nl");
     }
+    
+    else if (!strcmp(showParameter, "discord", true, 4)) {
+        showInfo = true;
+        format(g_message, sizeof(g_message), "Join us on Official LVP Discord Server at www.discord.sa-mp.nl ");
```

Comment:
1. Official is not spelled with a capital,
  1. It's discord.sa-mp.nl, no use of "www." in front it it,
  1. No space at the end.

"Join us on the official LVP Discord server at discord.sa-mp.nl"